tool/common/common.go

```diff
@@ -98,3 +101,34 @@ func ShowSessions(events []events.AuditEvent, format string, w io.Writer) error
 		return trace.BadParameter("unknown format %q", format)
 	}
 }
+
+// ShowClusterAlerts shows cluster alerts with the given label.
+func ShowClusterAlerts(ctx context.Context, client local.ClusterAlertGetter, w io.Writer, labels map[string]string, ignore map[string]string) error {
```

Comment:
Does this function have a unit test? If not, let's add it to test the filtering logic.